src/azul/plugins/metadata/anvil/indexer/transform.py

```diff
@@ -154,11 +154,9 @@ def estimate(self, partition: BundlePartition) -> int:
         return sum(map(partial(self._contains, partition), self.bundle.entities))
 
     def transform(self, partition: BundlePartition) -> Iterable[Contribution]:
-        return (
```

Comment:
Please remove this change. The expression signals that this is functional rather than imperative so I have to think less when looking at it. Size-wise this change is a wash.